### ISSUE TITLE
Cannot start mxnet-model-server on non-default port.

### ISSUE THREAD

**Issue by treske** [awslabs/mxnet-model-server#241]
Using the --port option for mxnet-model-server fails with the following error:
Failed to start model serving host: Flask handler failed to start: port must be an integer

Steps to recreate:

```bash
mkdir -p ~/env/mxnet-model-server-port-issue 
python3 -m venv ~/env/mxnet-model-server-port-issue
source ~/env/mxnet-model-server-port-issue/bin/activate
pip install mxnet-model-server    
cd ~/env/mxnet-model-server-port-issue
mxnet-model-server --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model --log-file ~/env/mxnet-model-server-port-issue/mms.log --log-level DEBUG --host 127.0.0.1 --port 8090
```

One way to fix would be to change line 135 in mms/mxnet_model_server.py

from
self.port = self.args.port or 8080
to
self.port = int(self.args.port) or 8080

**Comment by lupesko** [comment]
Thanks for finding the issue.
Feel free to contribute the fix, or I can push a fix in a day or two.

**Comment by yuruofeifei** [comment]
Fixed at #240
